public/components/event_analytics/explorer/save_panel/__tests__/__snapshots__/save_panel.test.tsx.snap

```diff
@@ -738,7 +738,7 @@ exports[`Saved query table component Renders saved query table 1`] = `
             onMouseOver={[Function]}
           >
             <EuiSwitch
-              checked={true}
+              checked={false}
```

Comment:
was this behavior not tested properly - this seems like a pretty big snapshot change?